experiments/openebs/openebs-nfs-provisioner-kill/openebs_nfs_provisioner_kill_ansible_logic.yml

```diff
@@ -0,0 +1,162 @@
+              c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+        when: lookup('env','CHAOSENGINE') 
+
+      ## RECORD START-OF-EXPERIMENT IN CHAOS RESULT CR
+      - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+        vars:
+          status: 'SOT'
+          namespace: "{{ a_ns }}"
+
+      ## DISPLAY APP INFORMATION 
+      - name: Display the app information passed via the test job
+        debug: 
+          msg: 
+            - "The application info is as follows:"
+            - "NFS Provisioner Namespace : {{ nfs_provisioner_ns }}"
+            - "NFS Provisioner LABEL     : {{ nfs_provisioner_label }}"
+            - "NFS Provisioner PVC       : {{ nfs_pvc }}"
+            - "NFS Service               : {{ nfs_svc }}"
+
+      ## RECORD EVENT FOR PRE-CHAOS CHECK
+      - include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
+        vars:
+          stage: "PreChaosCheck"
+          exp_pod_name: "{{ chaos_pod_name }}"
+          engine_ns: "{{ c_ns }}"
+          message: "OpenEBS NFS are Running successfully"
+        when: "c_engine is defined and c_engine != ''"
+
+      - block:
+          - name: Generate a external app run id
+            shell: echo $(mktemp) | cut -d '.' -f 2 
+            register: external_app_rand_string   
+
+          - set_fact:
+              external_app_run_id: "{{ external_app_rand_string.stdout | lower }}"
+
+          - name: Deploying and checking liveness of external application using NFS
+            include_tasks: "/utils/apps/nfs/external_liveness_check.yml"
+            vars:
+              app_ns: "{{ a_ns }}"
+              app_label: "{{ a_label }}"
+              state: "START"
+              run_id: "{{ external_app_run_id }}"
+
+        when: "external_app_check == 'true'"
+  
+      ## PRE-CHAOS APPLICATION LIVENESS CHECK
+      - name: Generate a nfs exporter liveness check run id
+        shell: echo $(mktemp) | cut -d '.' -f 2 
+        register: user_app_rand_string   
+
+      - set_fact:
+          user_app_run_id: "{{ user_app_rand_string.stdout | lower }}"
+
+      - name: (Pre) Verify that the AUT (Application Under Test) is running 
+        include_tasks: "/utils/common/status_app_pod.yml"
+        vars:
+          app_ns: "{{ a_ns }}"
+          app_label: "{{ a_label }}"      
+          delay: 2
+          retries: 90
+
+      - name: (Pre) Checking the user's pod volume is using NFS 
+        include_tasks: "/utils/apps/nfs/nfs_mount_check.yml"
+        vars:
+          app_ns: "{{ a_ns }}"
+          app_label: "{{ a_label }}"
+          run_id: "{{ user_app_run_id }}"
+          state: "START"
+
+      ## Injecting the NFS kill chaos
+      - name: Injecting the NFS provisioner container with {{ c_lib }} chaos c_lib
+        include_tasks: "{{ c_util }}"
+        vars:
+          namespace: "{{ nfs_provisioner_ns }}"
+          app_container: "{{ c_container }}"
+          a_label: "{{ nfs_provisioner_label }}"
```

Comment:
a_label var is already used for a different purpose in line 10, can we use a different name here?

Reply:
We can use it. for a_label, as nfs is also an application